themes/demo/layouts/default.htm

```diff
@@ -13,6 +13,7 @@
         <link rel="icon" type="image/png" href="{{ 'assets/images/winter.png'|theme }}">
         <link href="{{ 'assets/css/vendor.css'|theme }}" rel="stylesheet">
         <link href="{{ 'assets/css/theme.css'|theme }}" rel="stylesheet">
+        <link href="/modules/system/assets/css/snowboard.extras.css" rel="stylesheet">
```

Comment:
I could have sworn that including `extras` in the `{% framework %}` tag also included the stylesheet, at the very least the docs say that it does: https://wintercms.com/docs/ajax/extras

Reply:
You're right, it does haphazardly include it with the JS files here: https://github.com/wintercms/winter/blob/5b58103107d1e8f7f5caa9042d6496b676af2724/modules/cms/twig/FrameworkNode.php#L50

I feel it would be better to have the developer explicitly include it or not, in the case that they completely customise the styling of the extras functionality. Thoughts?

Reply:
Meh, the features added by the extras build are all UI based, so that combined with the fact that the current behaviour includes it makes me lean towards including it by default as well.

Reply:
No probs, I'll include it then.